settings/blt.settings.php

```diff
@@ -141,7 +141,7 @@
     // When developing locally, we use the host name to determine which site
     // factory site is active. The hostname must have a corresponding entry
     // under the multisites key.
-    $input = new ArgvInput($_SERVER['argv']);
+    $input = new ArgvInput(!empty($_SERVER['argv']) ? $_SERVER['argv'] : ['']);
```

Comment:
Would in be sufficient to default to `[]` rather than `['']`?

Reply:
The array we send into `ArgvInput` has to have at least one element. This is necessary because we have this line in `ArgvInput`'s constructor.

```
        // strip the application name
        array_shift($argv);
```

Sending an empty array results in another PHP warning.

Reply:
Looks good!